src/main/java/com/shopping_app/shoppingApp/Exceptions/UserAlreadyExist.java

```diff
@@ -0,0 +1,14 @@
+package com.shopping_app.shoppingApp.Exceptions;
+
+import org.springframework.http.HttpStatus;
+
+public class UserAlreadyExist extends BaseException {
```

Comment:
Can this not be handled in an existing exception class? Will we keep adding exception class for every entity in a similar way?

Reply:
yes we can avoid but to add custom exeception for all case will help us to esaily debug and later if we want to add any extra fields in this exception we can add also  , i guess..